DEPENDENCIES.md

```diff
@@ -125,3 +125,10 @@
 | [golang.org/x/sync@43a5402ce75a95522677f77c619865d66b8c57ab](https://cs.opensource.google/go) | BSD-3-Clause | N/A |
 | [golang.org/x/sys@1e4c9ba3b0c4fcddbe90893331bdc829813066a1](https://cs.opensource.google/go) | BSD-3-Clause | N/A |
 | [golang.org/x/text@v0.3.4](https://cs.opensource.google/go) | BSD-3-Clause | N/A |
+| [github.com/go-playground/validator/v10@v10.2.0](https://github.com/go-playground/validator) | MIT | N/A |
+| [github.com/go-playground/assert/v2@v2.0.1](https://github.com/go-playground/assert) | MIT | N/A |
+| [github.com/pelletier/go-toml/v2@v2.0.6](https://github.com/pelletier/go-toml) | MIT | N/A |
+| [github.com/goccy/go-json@v0.10.0](https://github.com/goccy/go-json) | MIT | N/A |
+| [github.com/leodido/go-urn@v1.2.1](https://github.com/leodido/go-urn) | MIT | N/A |
+| [github.com/go-playground/universal-translator@v0.18.0](https://github.com/go-playground/universal-translator) | MIT | N/A |
+| [github.com/go-playground/locales@v0.14.0](https://github.com/go-playground/locales) | MIT | N/A |
```

Comment:
I think these need to link to clearlydefined.io entries/CQs (also, strange to me that we're depending on `go-playground`...)

Reply:
well, the https://github.com/eclipse-che/che-machine-exec/actions/workflows/dependencies-md-check.yaml test is passing after several iterations to make it happy... but if you think these should be different links I can change them or you can suggest changes.

As to CQs, I checked the first onem couldn't find a CQ for that version, and decided since there's a whols bunch of other "N/A" entries, this might be acceptable. What do we do if there's no CQ?

Reply:
for example there's no toml/v2 in https://clearlydefined.io/?sort=releaseDate&sortDesc=true&name=go-toml :(

Reply:
I'm not too familiar with the dependencies check, but it may just be taking us at our word on "N/A" here. In the past, we've had to submit requests in clearlydefined to harvest license information for updated dependencies, though the UI is clunky at best. I'll try to take a look today.